src/content/index.md

```diff
@@ -30,12 +30,15 @@ export default function bar() {
 
 <div class="splash__wrap">
 <div class="splash__left">
-__webpack.config.js__
+__Without config__ or provide custom __webpack.config.js__
```

Comment:
Can we clean `__` from `__Without config__`?
I think we only use them with file names.

Reply:
It is for giving a strong emphasis on 0cjs, we need @TheLarkInn to have a look at this

Reply:
Sure! We should give emphasis on ***0CJS***. Not sure if `__` is the best way to do it.